--- lxsession-0.4.6.2/configure.ac.omv~	2014-02-04 21:48:32.367148075 +0100
+++ lxsession-0.4.6.2/configure.ac	2014-02-04 21:48:45.573814324 +0100
@@ -156,10 +156,10 @@ if test x"$enable_man" = x"yes"; then
 	fi
 
 	dnl check for DocBook DTD and stylesheets in the local catalog.
-	JH_CHECK_XML_CATALOG([-//OASIS//DTD DocBook XML V4.1.2//EN],
-		[DocBook XML DTD V4.1.2], [], enable_man=no)
-	JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl],
-		[DocBook XSL Stylesheets >= 1.70.1], [], enable_man=no)
+dnl	JH_CHECK_XML_CATALOG([-//OASIS//DTD DocBook XML V4.1.2//EN],
+dnl		[DocBook XML DTD V4.1.2], [], enable_man=no)
+dnl	JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl],
+dnl		[DocBook XSL Stylesheets >= 1.70.1], [], enable_man=no)
 fi
 
 AM_CONDITIONAL(ENABLE_REGENERATE_MAN, test "x$enable_man" != "xno")
